sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ai/inference/azure_telemetry_instrumentor.py

```diff
@@ -0,0 +1,20 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from abc import ABC, abstractmethod  
+  
+class AzureTelemetryInstrumentor(ABC):
```

Comment:
no strong opinion, but do we really need it?

Reply:
i'd leave in case we add other instrumentors in the future

Reply:
I don't agree that it is better approach to add a common interface later but i can remove this for now.